src/config/env.ts

```
@@ -16,9 +16,19 @@ const envSchema = z.object({
   JWT_EXPIRES_IN: z.string().default("7d"),
   JWT_CLOCK_TOLERANCE_SECONDS: z.coerce.number().default(30),
   API_KEY_SALT: z.string().default(""),
-  ADMIN_API_KEY: z.string().optional(),
   RATE_LIMIT_WINDOW_MS: z.coerce.number().default(60000),
   RATE_LIMIT_MAX_REQUESTS: z.coerce.number().default(100),
+  // B-058: 64-char hex key (32 bytes) for AES-256-GCM PII field encryption.
+  // Generate with: node -e "console.log(require('crypto').randomBytes(32).toString('hex'))"
+  PII_ENCRYPTION_KEY: z
+    .string()
+    .length(64, "PII_ENCRYPTION_KEY must be exactly 64 hex characters (32 bytes)")
```

ESLint

[error] Replace `64,·"PII_ENCRYPTION_KEY·must·be·exactly·64·hex·characters·(32·bytes)"` with `⏎······64,⏎······"PII_ENCRYPTION_KEY·must·be·exactly·64·hex·characters·(32·bytes)",⏎····`

(prettier/prettier)